### PR TITLE
Add new question variants for the CE schema

### PR DESCRIPTION
**Pull request by berroar**
### Context
This PR has been raised to add question variants for the `live-in-establishment` and `number-of-visitors-in-establishment` question in the Communal Establishment schema.

### How to review
Check that the changes match and follow the routing of the overflow diagram attached to the trello card.

### CODE REVIEW

File: scripts/run_validator.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-branch=latest
```
Comment (berroar) [inline scripts/run_validator.sh:3]: Revert before merge

**Comment by berroar** [comment]
[[D] CE form - new question variant (S)](https://trello.com/c/h4myjeCB/3837-d-ce-form-new-question-variant-s)

File: source/jsonnet/england-wales/communal-establishment/blocks/establishment-details/people_in_establishment.jsonnet [hunk 2]
```diff
@@ -1,28 +1,98 @@
...
+      when: [
         {
-          description: '<em>Include</em> everyone from the groups you selected for the previous question',
+          id: 'live-in-establishment-answer',
+          condition: 'set',
         },
+      ],
+    },
+    {
+      question: questionWithExclusive,
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/communal-establishment/blocks/establishment-details/people_in_establishment.jsonnet:76]: As discussed, the exclusive question needs to be the first variant. As there is a bug with only the first variant being used when generating the submission, the exclusive answer will never appear in the downstream data.

Comment (berroar) [reply source/jsonnet/england-wales/communal-establishment/blocks/establishment-details/people_in_establishment.jsonnet:76]: ah, I forgot to swap them

File: scripts/run_validator.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
+branch=mutually-exclusive-variants
```
Comment (MebinAbraham) [inline scripts/run_validator.sh:3]: Reminder to reevert.

File: source/jsonnet/england-wales/communal-establishment/blocks/establishment-details/number_of_visitors_in_establishment.jsonnet [hunk 4]
```diff
@@ -27,8 +27,74 @@ local question = {
...
+      question: questionWithExclusive,
+      when: [
+        {
+          id: 'visitors-in-establishment-answer',
+          condition: 'not set',
+        },
+        {
+          id: 'visitors-in-establishment-exclusive',
+          condition: 'not set',
+        },
```
Comment (MebinAbraham) [inline source/jsonnet/england-wales/communal-establishment/blocks/establishment-details/number_of_visitors_in_establishment.jsonnet:87]: This shouldn't be needed as it has to be `not set` to get to this point.

File: source/jsonnet/england-wales/communal-establishment/blocks/establishment-details/people_in_establishment.jsonnet [hunk 5]
```diff
@@ -1,28 +1,98 @@
...
         {
-          description: '<em>Include</em> everyone from the groups you selected for the previous question',
+          id: 'live-in-establishment-answer',
+          condition: 'not set',
         },
         {
-          description: '<em>Do not include</em> visitors who have another UK address, or usually live outside the UK and intend to stay in the UK for 3 months or less',
+          id: 'live-in-establishment-exclusive',
+          condition: 'not set',
         },
```
Comment (MebinAbraham) [inline source/jsonnet/england-wales/communal-establishment/blocks/establishment-details/people_in_establishment.jsonnet:85]: Same with this, this has to be `set` by this point.

**Pull request merged**